styles/style.css

```diff
@@ -0,0 +1,333 @@
+  border: solid 1px;
+  border-color: ;
+  border-radius:;
+  background:;
+}
+
+.why {
+  grid-row-start: 4;
+  margin-top: 60px;
+  margin-left: 15px;
+  margin-right: 15px;
+  border-bottom: 1px ;
+}
+
+.why .container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.why ul {
+  display: flex;
+  flex-direction: row;
+}
+
+.why li {
+  margin: 20px;
+  list-style: none;
+  width: 33%;
+  display: grid;
+  grid-template-rows: 20px 1fr;
+  grid-template-columns: 114px 1fr;
+}
+
+.why li strong {
+  grid-column-start: 2;
+  grid-row-start: 1;
+  display: block;
+}
+
+.why li p {
+  grid-column-start: 2;
+  grid-row-start: 2;
+  display: block;
+}
+
+.quote {
+  grid-row-start: 3;
+  border-top: ;
+  border-bottom: ;
+  margin-left: 15px;
+  margin-right: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-top: 3em;
+  padding-bottom: 3em;
+  border-bottom: solid black;
+  border-bottom-width: .05em;
+}
+
+blockquote {
+  font-size: 1.5em;
+  font-style: italic;
+  line-height: 1.5em;
+  margin-left: 20%;
+  margin-right: 20%;
+  quotes: """ """;
+}
+
+blockquote:before {
+  content: open-quote;
+}
+
+blockquote:after {
+  content: close-quote;
+}
+
+.quote blockquote {
+}
+
+blockquote + p {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 20%;
+  font-size: 1.5em;
+  font-style: italic;
+}
+
+blockquote + p::before {
+  content: "--";
+  margin-right: 6px;
+}
+
+.second-quote {
+  grid-row-start: 5;
+  border-bottom: 1px;
+  margin-left: 15px;
+  margin-right: 15px;
+  margin-top: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-top: 3em;
+  padding-bottom: 3em;
+  border-bottom: solid black;
+  border-bottom-width: .05em;
+}
+
+.team {
+  grid-row-start: 6;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 35px;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.team ul {
+  margin-left: -36px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  list-style: none;
+}
+
+.top-pics {
+  display: flex;
+  flex-direction: row;
+  margin-top: 15px;
+}
+
+.bottom-pics {
+  display: flex;
+  flex-direction: row;
+}
+
+.team li {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.team li {
+  margin-bottom: 30px;
+}
+
+.name {
+  margin-top: 10px;
+}
+
+.team img {
+  border-radius: 50%;
+  margin-left: 40px;
+  margin-right: 40px;
+  padding: 5px;
+  border: 2px;
+}
+
+.sponsors {
+  grid-row-start: 7;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-bottom: 1px ;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.sponsors ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+  list-style: none;
+  margin-top: 25px;
+}
+
+.sponsors li {
+  margin: 10px;
+}
+
+
+.footer {
+  grid-row-start: 8;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
```

Comment:
For some reason, the li in "About" are indented and those in "Support" aren't. Not totally sure why, but might be worth playing around with!